x-pack/plugins/actions/server/builtin_action_types/email.ts

```diff
@@ -36,6 +36,7 @@ const ConfigSchemaProps = {
   port: schema.nullable(portSchema()),
   secure: schema.nullable(schema.boolean()),
   from: schema.string(),
+  hasAuth: schema.boolean({ defaultValue: false }),
```

Comment:
I wonder if this should be `true` to reflect the UI or provide no default value here(for the API level).

```suggestion
  hasAuth: schema.boolean({ defaultValue: true }),
```